features/support/commit_helpers.rb

```diff
@@ -138,5 +144,12 @@ def verify_commits commits_array
 
   actual_commits = commits_in_repo
 
+  # Leave only the expected keys in actual_commits
+  unless expected_commits[0].key? :file_content
+    actual_commits.each do |commit_data|
+      commit_data.delete :file_content
+    end
+  end
```

Comment:
How about just updating `normalize_expected_commit_data` to add the `file_content` if its not already there?